acme/setup.py

```diff
@@ -22,15 +22,14 @@
 ]
 
 # env markers in extras_require cause problems with older pip: #517
+# Keep in sync with conditional_requirements.py.
```

Comment:
It is a bit scary. My favored approach (please see https://github.com/letsencrypt/letsencrypt/pull/2182#issuecomment-172099220 and the following comment) is to do away with conditional dependencies.